programs/CMakeLists.txt

```diff
@@ -47,8 +47,8 @@ if(LIBDEFLATE_BUILD_GZIP)
         set(GZIP ${CMAKE_INSTALL_FULL_BINDIR}/libdeflate-gzip)
         set(GUNZIP ${CMAKE_INSTALL_FULL_BINDIR}/libdeflate-gunzip)
         install(CODE "message(\"-- Installing: \$ENV{DESTDIR}${GUNZIP}\")")
-        install(CODE "file(CREATE_LINK \$ENV{DESTDIR}${GZIP}
-                      \$ENV{DESTDIR}${GUNZIP})" COPY_ON_ERROR)
+        install(CODE "file(CREATE_LINK \$ENV{DESTDIR}${GZIP}${CMAKE_EXECUTABLE_SUFFIX}
+                      \$ENV{DESTDIR}${GUNZIP}${CMAKE_EXECUTABLE_SUFFIX})" COPY_ON_ERROR)
```

Comment:
I'm wondering if the `COPY_ON_ERROR` should be part of `file()`?

Reply:
Yes, I think so.  I messed this up, sorry.

Reply:
I agree, I will fix it in this PR.